frontend/src/ReusableButton/ReusableButton.css

```diff
@@ -0,0 +1,13 @@
+.button {
+    font-size: 16px;
+    padding: 10px 20px;
+    border-radius: 5px;
+    transition: background-color 0.3s ease, border-color 0.3s ease;
+  }
+  
+  /* Custom hover effect */
+  .button:hover {
+    background-color: #0056b3;
+    border-color: #004085;
```

Comment:
Hover effect chosen colors does not seems to meet accessibility criterias
https://webaim.org/resources/contrastchecker/

Reply:
Chosen colors are just a starting point, colors for project have not been chosen yet